code/starfield/starfield.cpp

```diff
@@ -468,16 +472,8 @@ void parse_startbl(const char *filename)
 				stuff_float(&sbm.b);
 				stuff_float(&sbm.i);
 
-				if (optional_string("$SunSpecularRGB:")) {
-					stuff_float(&sbm.spec_r);
-					stuff_float(&sbm.spec_g);
-					stuff_float(&sbm.spec_b);
-				}
-				else {
-					sbm.spec_r = sbm.r;
-					sbm.spec_g = sbm.g;
-					sbm.spec_b = sbm.b;
-				}
+				if (optional_string("$SunSpecularRGB:"))
+					Warning(LOCATION, "Sun %s tried to set SunSpecularRGB. This feature has been deprecated and will be ignored.", sbm.filename);
```

Comment:
I am not sure how often this is used, but I think this can be downgraded to a log print instead of a Warning